src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -97,6 +99,9 @@ object SbtGit extends Plugin {
     gitCurrentBranch in ThisBuild <<= (gitReader in ThisBuild) apply { (reader) =>
       // TODO - Make current branch an option?
       Option(reader.withGit(_.branch)) getOrElse ""
+    },
+    gitUncommittedChanges in ThisBuild <<= (gitReader in ThisBuild) apply { (reader) =>
```

Comment:
you may want to check out my branch of 0.8.  We're dropping the 0.12 syntax for 0.13.